parcels/fieldfilebuffer.py

```diff
@@ -9,14 +9,22 @@
 from dask import utils as da_utils
 from netCDF4 import Dataset as ncDataset
 
+from parcels._typing import InterpMethodOption
```

Comment:
Why do we call this `InterpMethodOption` and not `InterpMethod`? The other types also don't end in `...Option`

Reply:
This is highlighted in the `_typing.py` file. `InterpMethod` also includes typing for dictionary mappings